privval/file_test.go

```diff
@@ -321,3 +359,15 @@ func newProposal(height int64, round int32, blockID types.BlockID, t time.Time)
 		Timestamp: t,
 	}
 }
+
+func newTestFilePV(t *testing.T) (*FilePV, string, string) {
```

Comment:
I don't see the 2nd and 3rd args being used, I don't think we should return them if they aren't used.

Reply:
They're being used here: https://github.com/tendermint/tendermint/pull/8141/files#diff-8881ac9666ab8aa476a419622ae008801c46bfb7693cc47a807fd7bdf32e12a9R24

and here: https://github.com/tendermint/tendermint/pull/8141/files#diff-8881ac9666ab8aa476a419622ae008801c46bfb7693cc47a807fd7bdf32e12a9R41